leftwm-core/src/models/focus_manager.rs

```diff
@@ -33,6 +33,23 @@ impl FocusBehaviour {
     }
 }
 
+/// Controls behaviour for window activation. Default is to mark the window as urgent.
+#[derive(Serialize, Deserialize, Debug, Clone, Copy, PartialEq, Eq)]
+pub enum FocusOnActivationBehaviour {
+    /// Do nothing.
+    DoNothing,
+    /// Mark the window as urgent.
+    MarkUrgent,
+    /// Switch to the window.
+    SwitchTo,
+}
+
+impl Default for FocusOnActivationBehaviour {
+    fn default() -> Self {
+        Self::MarkUrgent
+    }
+}
```

Comment:
How about using the `Default` macro with `#[default]` over the type instead? (You can do it for `FocusBehaviour` above too if you want)

Reply:
Sure, that probably should have been done in the first place so I'll fix up both.